src/ui/CategoryFacet/CategoryFacetTemplates.ts

```diff
@@ -49,11 +50,18 @@ export class CategoryFacetTemplates {
     return this.collapseArrow.clone(true);
   }
 
+  private getFormattedCount(count: number) {
+    if (count > 0) {
```

Comment:
You could use a ternary here make the code more elegant. e.g.
```
private getFormattedCount(count: number) {
    return count > 0 ? Globalize.format(count, 'n0') : count;
}
```

For the condition, `count > 0`, I am curious why we do not globalize `0`? Do we need an exception for that number?

Reply:
Actually, I was following what I saw in FacetValue. 
I'm happy to remove the condition.

Reply:
I updated to remove the condition on `count`